zerver/migrations/0378_message_edit_history_update_fields.py

```diff
@@ -0,0 +1,81 @@
+import time
+
+import orjson
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import Min, Model
+
+BATCH_SIZE = 1000
+
+
+def backfill_message_edit_history(first_id: int, last_id: int, Message: Model) -> None:
+    messages = Message.objects.filter(edit_history__isnull=False, id__range=(first_id, last_id))
```

Comment:
I added this in my revision so that now the four columns that are fetched from the Message table are: "subject", "recipient", "edit_history", "edit_history_update_fields".